lib/NagiosStatusConnector.php

```diff
@@ -11,17 +14,15 @@
  */
 class NagiosStatusConnector extends StatusConnector
 {
-    const NAGIOS_URL = 'status.nagios.url';
-    const NAGIOS_CERT_PATH = 'status.nagios.certificate_path';
-    const NAGIOS_CERT_PASSWORD = 'status.nagios.certificate_password';
-    const NAGIOS_CA_PATH = 'status.nagios.ca_path';
-    const NAGIOS_PEER_VERIFY = 'status.nagios.peer_verification';
+    const NAGIOS_HOST = 'status.nagios.host';
+    const NAGIOS_KEY_PATH = 'status.nagios.key_path';
+    const NAGIOS_LOGIN = 'status.nagios.login';
+    const NAGIOS_COMMAND = 'status.nagios.command';
```

Comment:
Lately, we were discussing switching to a configuration consisting of blocks. I would instead of using the "." separated properties create a block structure to hold the configuration.